src/cf-forms/usage.md

```diff
@@ -22,39 +22,16 @@ Capital Framework.
     - [Label helper text](#label-helper-text)
 - [Inputs](#inputs)
     - [Basic Text Inputs](#basic-text-inputs)
-        - [Default state](#default-state)
-        - [Hover state](#hover-state)
-        - [Focus state](#focus-state)
-        - [Disabled state](#disabled-state)
         - [Full-width modifier](#full-width-modifier)
     - [Basic checkboxes](#basic-checkboxes)
-        - [Default state](#default-state)
-        - [Hover/focus state](#hoverfocus-state)
-        - [Selected state](#selected-state)
-        - [Disabled state](#disabled-state)
     - [Basic radio buttons](#basic-radio-buttons)
-        - [Default state](#default-state)
-        - [Hover/focus state](#hoverfocus-state)
-        - [Selected state](#selected-state)
-        - [Disabled state](#disabled-state)
     - [Large target area checkboxes](#large-target-area-checkboxes)
-        - [Default state](#default-state)
-        - [Hover/focus state](#hoverfocus-state)
-        - [Selected state](#selected-state)
-        - [Disabled state](#disabled-state)
     - [Large target area radio buttons](#large-target-area-radio-buttons)
-        - [Default state](#default-state)
-        - [Hover/focus state](#hoverfocus-state)
-        - [Selected state](#selected-state)
-        - [Disabled state](#disabled-state)
 - [Buttons](#buttons)
     - [Simple input with a button](#simple-input-with-a-button)
     - [Button inside an input](#button-inside-an-input)
     - [Button inside an input with a button](#button-inside-an-input-with-a-button)
 - [Select dropdown](#select-dropdown)
-    - [Default state](#default-state)
-    - [Hover/focus state](#hoverfocus-state)
-    - [Disabled state](#disabled-state)
 - [Basic multiselect](#basic-multiselect)
 - [Form fieldsets](#form-fieldsets)
```

Comment:
Since the whole page is forms I suggest "Fieldsets". This is consistent with the DM. In the DM we refer to the "Fieldset legend." Here in CF we show the "Legend" far up the page and the "Fieldset" at the very bottom. Does the "Legend" get used in any circumstance other than with the "Fieldset"? If so should they appear closer together on the CF docs page?